__test__/components/kytos/inputs/buttons/Button.test.js

```diff
@@ -0,0 +1,96 @@
+import { mount, shallowMount } from '@vue/test-utils';
+import Button from '@/components/kytos/inputs/buttons/Button.vue';
+import { describe, test, expect, beforeAll, afterEach, vi } from "vitest";
+
+
+
+describe("Button.vue", () => {
+    let wrapper;
+    beforeAll(() => {
+        expect(Button).toBeTruthy();
+    });
+
+    afterEach(() => {
+        wrapper.unmount();
+        vi.restoreAllMocks();
+    });
+
+    //Inputs
+
+    describe("Props", () => {
+        test("Disabling Button", async () => {
+            wrapper = mount(Button);
+            expect(wrapper.exists()).toBe(true);
+            const mainButton = wrapper.get('[data-test="main-button"]');
+            expect(mainButton.element.hasAttribute('disabled')).toBe(false);
+
+            await wrapper.setProps({ isDisabled: true });
+
+            expect(mainButton.element.hasAttribute('disabled')).toBe(true);
+        });
+
+        test("Button Tooltip", async () => {
+            const testValue = 'test';
+            wrapper = mount(Button);
+            expect(wrapper.exists()).toBe(true);
+            const mainButton = wrapper.get('[data-test="main-button"]');
+
+            expect(mainButton.element.hasAttribute('title')).toBe(false);
+
+            await wrapper.setProps({ tooltip: testValue });
+
+            expect(mainButton.attributes('title')).toBe(testValue);
+        });
+    });
+
+    describe("User Interactions", () => {
+        test("Button Data/Write/Use", async () => {
```

Comment:
Could you also add another test to cover this case:

- Clicking on a disabled button it won't emit `click`